src/pretix/plugins/sendmail/templates/pretixplugins/sendmail/rule_inspect.html

```diff
@@ -45,7 +45,7 @@ <h1>{% trans "Inspect Email Rule" %}</h1>
                     {% if request.event.has_subevents %}
                         <td>
                             <a href="{% url "control:event.subevent" organizer=request.event.organizer.slug event=request.event.slug subevent=sm.subevent.id %}?returnto={{ request.GET.urlencode|urlencode }}">
-                                {{ sm.subevent.name }}
+                                {{ sm.subevent.name }} - {{ sm.subevent.date_from }}
                             </a><br>
                             {{ sm.get_date_range_display }}
```

Comment:
should be sm.subevent.get_date_range_display :)

Reply:
Ah, of course! :)